test/it/uk/gov/hmrc/exports/repositories/SubmissionRepositorySpec.scala

```diff
@@ -54,7 +51,7 @@ class SubmissionRepositorySpec
         repo.save(submission).futureValue must be(submission)
 
         val submissionInDB = repo.findSubmissionByMrn(mrn).futureValue
-        submissionInDB must be(defined)
+        submissionInDB must be('defined)
```

Comment:
What is the purpose of the apostrophies in this file?

Reply:
For some reason Intellij marks `defined` with a missing implicit when the apostrophe is not added. The test is working in both cases still.  Not a big deal btw. I can remove it if the apostrophe only adds noise.